### DISCUSSION TITLE
使用Discoed评论通知出错

### DISCUSSION THREAD

**Discussion by StudyingLover** [walinejs/waline#1775] [Q&A]
我按照https://waline.js.org/guide/features/notification.html#%E9%99%84%E5%8A%A0%E8%AF%B4%E6%98%8E配置了我的vercel，然后我使用时发现当我提交评论是会出现如下的问题
![image](https://user-images.githubusercontent.com/68844494/215322885-fc5e17a8-b0ad-485c-a065-715dcc5b2ab8.png)
马赛克涂掉的是我的discord webhook URL
我的通知模板是
```
{{site.name|safe}} 有新评论啦
{{self.nick}} 评论道:
{{self.comment}}
邮箱: {{self.mail}}
状态: {{self.status}}
仅供评论预览，查看完整內容:
{{site.postUrl}}
```
我希望得到解决方案

**Comment by Fity** [comment]
`Discord` 默认 `webhook` 是异步通知，接口响应是一个 `http 204 no content`。
要拿到正确的 `json` 结果需要把通知改成同步。即，在 `webhook` 请求地址上添加 `wait=true` 参数。

---
`Discord` default `webhook` is an asynchronous notification, and the interface response is a `http 204 no content`.
To get the correct `json` result, you need to change the notification to synchronous. That is, add the `wait=true` parameter to the `webhook` request address.

**Comment by StudyingLover** [comment]
可以给出链接的一个例子吗
---
Can you give an example of the link

**Reply by Fity** [reply]
就正常的 `query string` 参数啊。原来的 `url` 是 `xxx.com` 的话，你就改成 `xxx.com?wait=true` 。已经有参数了的话（`xxx.com?aaa=bbb`），就改成 `xxx.com?aaa=bbb&wait=true`

**Reply by StudyingLover** [reply]
懂了，感谢

---
Understood, thanks
